docs/docs/tutorial/tpch.md

```diff
@@ -159,12 +159,579 @@ LOAD DATA LOCAL INFILE '/home/username/workspace/tpch1g/lineitem/lineitem.tbl' I
+  "c_comment"     VARCHAR(117),
+  "c_dummy"       VARCHAR(10),
+  PRIMARY KEY ("c_custkey"));
+
+-- part
+CREATE TABLE IF NOT EXISTS "tpch1g"."part" (
+  "p_partkey"     INT,
+  "p_name"        VARCHAR(55),
+  "p_mfgr"        CHAR(25),
+  "p_brand"       CHAR(10),
+  "p_type"        VARCHAR(25),
+  "p_size"        INT,
+  "p_container"   CHAR(10),
+  "p_retailprice" DECIMAL(15,2) ,
+  "p_comment"     VARCHAR(23) ,
+  "p_dummy"       VARCHAR(10),
+  PRIMARY KEY ("p_partkey"));
+
+-- partsupp
+CREATE TABLE IF NOT EXISTS "tpch1g"."partsupp" (
+  "ps_partkey"     INT,
+  "ps_suppkey"     INT,
+  "ps_availqty"    INT,
+  "ps_supplycost"  DECIMAL(15,2),
+  "ps_comment"     VARCHAR(199),
+  "ps_dummy"       VARCHAR(10),
+  PRIMARY KEY ("ps_partkey"));
+
+-- orders
+CREATE TABLE IF NOT EXISTS "tpch1g"."orders" (
+  "o_orderkey"       INT,
+  "o_custkey"        INT,
+  "o_orderstatus"    CHAR(1),
+  "o_totalprice"     DECIMAL(15,2),
+  "o_orderdate"      DATE,
+  "o_orderpriority"  CHAR(15),
+  "o_clerk"          CHAR(15),
+  "o_shippriority"   INT,
+  "o_comment"        VARCHAR(79),
+  "o_dummy"          VARCHAR(10),
+  PRIMARY KEY ("o_orderkey"));
+
+-- lineitem
+CREATE TABLE IF NOT EXISTS "tpch1g"."lineitem"(
+  "l_orderkey"    INT,
+  "l_partkey"     INT,
+  "l_suppkey"     INT,
+  "l_linenumber"  INT,
+  "l_quantity"    DECIMAL(15,2),
+  "l_extendedprice"  DECIMAL(15,2),
+  "l_discount"    DECIMAL(15,2),
+  "l_tax"         DECIMAL(15,2),
+  "l_returnflag"  CHAR(1),
+  "l_linestatus"  CHAR(1),
+  "l_shipdate"    DATE,
+  "l_commitdate"  DATE,
+  "l_receiptdate" DATE,
+  "l_shipinstruct" CHAR(25),
+  "l_shipmode"    CHAR(10),
+  "l_comment"     VARCHAR(44),
+  "l_dummy"       VARCHAR(10));
+```
+
+### Import Data
+
+Suppose your work directory is `/home/username/workspace` and the tpch1g data is stored in `/home/username/workspace/tpch1g`. Then, issue the following commands in the PostgreSQL prompt to load the data.
+
+```bash
+\copy "tpch1g"."region"     from '/home/username/workspace/tpch1g/region/region.tbl'        DELIMITER '|' CSV;
+\copy "tpch1g"."nation"     from '/home/username/workspace/tpch1g/nation/nation.tbl'        DELIMITER '|' CSV;
+\copy "tpch1g"."customer"   from '/home/username/workspace/tpch1g/customer/customer.tbl'    DELIMITER '|' CSV;
+\copy "tpch1g"."supplier"   from '/home/username/workspace/tpch1g/supplier/supplier.tbl'    DELIMITER '|' CSV;
+\copy "tpch1g"."part"       from '/home/username/workspace/tpch1g/part/part.tbl'            DELIMITER '|' CSV;
+\copy "tpch1g"."partsupp"   from '/home/username/workspace/tpch1g/partsupp/partsupp.tbl'    DELIMITER '|' CSV;
+\copy "tpch1g"."orders"     from '/home/username/workspace/tpch1g/orders/orders.tbl'        DELIMITER '|' CSV;
+\copy "tpch1g"."lineitem"   from '/home/username/workspace/tpch1g/lineitem/lineitem.tbl'    DELIMITER '|' CSV;
+```
 
 ## Apache Spark
 
+### Create table and load data
+
+Simply copy and paste following queries to spark to set up TPC-H tables.
```

Comment:
Please include the HDFS uploading step. This approach---referencing the local files from Spark---does not work in general.